runelite-client/src/main/java/net/runelite/client/plugins/regenmeter/RegenMeterPlugin.java

```diff
@@ -149,5 +154,12 @@ else if (currentHP > maxHP)
 			// Show it going down
 			hitpointsPercentage = 1 - hitpointsPercentage;
 		}
+		else if (!messageSend && maxHP > currentHP && config.regenWarning() > 0 && config.regenWarning() < 100 && ((Math.round(hitpointsPercentage * 100)) == config.regenWarning() || (Math.round(hitpointsPercentage * 100)) == config.regenWarning() - 1))
+		{
+			notifier.notify("You have reached your HP Regeneration Threshold!");
```

Comment:
You should probably follow the notification pattern in idle notifier here

Reply:
I am not a 100% sure what you meant with this but i think my last commit fixes the issue you had with it?